test.sh

```diff
@@ -6,12 +6,26 @@
 # TEST_OPENSHIFT_MODE - If set, run OpenShift tests (if present)
 # VERSIONS - Must be set to a list with possible versions (subdirectories)
 
-set -eE
-
 trap 'echo "errexit on line $LINENO, $0" >&2' ERR
```

Comment:
Also this line no longer has any effect.

Reply:
https://github.com/sclorg/container-common-scripts/pull/369/commits/2229acbe720e89bae02bb70256af4efca888e5d7